### ISSUE TITLE
[BUG] Paragraphs are missing 16px margin not aligning with UI guidelines 

### ISSUE THREAD

**Issue by simonequach** [kirbydesign/designsystem#3429]
## Describe the bug
Paragraphs are missing 16px margin not aligning with UI guidelines. Leaving it up to every team to manually add it, potentially causing problems and a lot of UI debt on the long run, if changes in the UI guidelines are made. 

## What was the expected behavior?
That a 16px margin is default, aligning text with headers as required in the UI guidelines. 

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

**Comment by jakobe** [comment]
@simonequach We need more info on this: On which side(s) do you want the margin? Top/Bottom? Left/Right?
Would be great if you could add a screenshot of the changes you would like to see 🤞🏻

**Comment by simonequach** [comment]
@jakobe it's missing on the lefthand side, so headers and paragraphs doesnt align. Henrik is aware of this. 
Every team fix it manually, so I do not have a photo to show.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
